pom.xml

```diff
@@ -800,6 +800,12 @@
             <scope>provided</scope>
             <!-- License: Apache 2.0 -->
          </dependency>
+         <dependency>
```

Comment:
Guava is only used by airlift.. Do we really need to add the dependency globally?

Reply:
I see your point but airlift/airline is added as global dependency and Guava is used by artemis-cli (XmlDataExporterUtil), by integrations-tests (MethodCalledVerifier) by airlift/airline and by hawtio/web.
Airlift/airline is added as global dependency, as artemis-cli dependency and as artemis-distribution dependency.
Hawtio/web is added as artemis-console dependency.